windows/jitsi-meet/render.js

```diff
@@ -1,40 +1,35 @@
-/* global process */
-const utils = require("jitsi-meet-electron-utils");
+/* global process, JitsiMeetExternalAPI */
 const {
     RemoteControl,
     setupScreenSharingForWindow
-} = utils;
-const config = require("../../config.js");
+} = require("jitsi-meet-electron-utils");
+const { jitsiMeetDomain } = require("../../config.js");
 
 /**
- * The remote control instance.
+ * Loads a script from a specific source.
+ *
+ * @param src the source from the which the script is to be (down)loaded
+ * @param loadCallback on load callback function
+ * @param errorCallback callback to be called on error loading the script
  */
-let remoteControl;
+function loadScript(
+        src,
+        loadCallback = () => {},
+        errorCallback = console.error) {
+    const script = document.createElement('script');
 
-/**
- * Cteates the iframe that will load Jitsi Meet.
- */
-let iframe = document.createElement('iframe');
-iframe.src = process.env.JITSI_MEET_URL || config.jitsiMeetURL;
-iframe.allowFullscreen = true;
-iframe.onload = onload;
-document.body.appendChild(iframe);
+    script.async = true;
 
-/**
- * Handles loaded event for iframe:
- * Enables screen sharing functionality to the iframe webpage.
- * Initializes postis.
- * Initializes remote control.
- */
-function onload() {
-    setupScreenSharingForWindow(iframe.contentWindow);
-    iframe.contentWindow.onunload = onunload;
-    remoteControl = new RemoteControl(iframe);
+    script.onload = loadCallback;
+    script.onerror = errorCallback;
+    script.src = src;
+    document.head.appendChild(script);
 }
 
-/**
- * Clears the postis objects and remoteControl.
- */
-function onunload() {
-    remoteControl.dispose();
-}
+loadScript(`https://${jitsiMeetDomain}/external_api.js`, () => {
```

Comment:
This way we make sure that the script will be loaded from the same deployment specified in the config.js. 

For me it is very useful for testing. But I think it will be useful in general too.

If somebody has it's own deployment and want to use the app with it probably will want to load the external_api.js from the same place.

Do you have any other suggestions? We may want to add another config option only for the script but I think this is a little bit too much for now? Or we can put 'meet.jit.si' there but if somebody want to change it, I think it won't be very convenient?